client/js/table-view.js

```diff
@@ -74,10 +76,50 @@ class TableView{
         this.formulaBarEl.addEventListener('keyup', this.handleFormulaBarChange.bind(this));
     }
 
+    renderSum(collie, total) {
+        const whichColumn = document.querySelector(`#sum-${collie}`);
```

Comment:
It's brittle (breaky) to have to maintain all these id numbers. 

If you really want to pluck out a column sum cell by its numeric index, why not just do `document.querySelectorAll()` on a CSS class name or even better just by a tag name selector (like `TFOOT TD` or something), convert the result to an array, and index into that array?  

There actually is a reasonable counterargument to that, but I'm interested in your thinking.